spec/core-spec.js

```diff
@@ -165,4 +165,48 @@ describe("CORE", function() {
 			expect(ticks.size()).to.be.equal(0);
 		});
 	});
+
+	describe("options", () => {
+		before(() => {
+			args = {
+				data: {
+					columns: [
+						["data1", 30, 200, 100, 50, 120, 175],
+						["data2", 130, 100, 0, 170, 75, 140]
+					]
+				},
+				clipPath: true
+			};
+		});
+
+		it("clip-path property should be null", () => {
```

Comment:
this test case is checking about chart element has clip-path attribute, so to change title like "chart should have clip-path property"